src/bll/reducer/actionConstant.ts

```diff
@@ -0,0 +1,15 @@
+enum ActionConstant {
```

Comment:
Enum is not a constant.
```suggestion
enum Actions {
```